starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/aotDependencies.rocker.raw

```diff
@@ -6,6 +6,7 @@
                   <dependency>
                       <groupId>@(aotDependency.getGroupId())</groupId>
                       <artifactId>@(aotDependency.getArtifactId())</artifactId>
+                      <version>${micronaut.security.version}</version>
```

Comment:
we have to pass this as a parameter to the rocker template. If we have another module contribute an AOT optimisations. e.g. aws, we will need `${micronaut.aws.version}`.

Reply:
🤦 

Done in https://github.com/micronaut-projects/micronaut-starter/pull/2025/commits/70d61fddf7db919146ce062f81612ccc72bb9a8d